Source/WebKit/Shared/RemoteLayerTree/CGDisplayListImageBufferBackend.mm

```diff
@@ -79,9 +79,10 @@ void setCTM(const WebCore::AffineTransform& transform) final
         return m_immutableBaseTransform * GraphicsContextCG::getCTM(includeDeviceScale);
     }
 
-    WebCore::FloatRect roundToDevicePixels(const WebCore::FloatRect& rect, RoundingMode = RoundAllSides) const final
+    std::optional<std::pair<float, float>> scaleForRoundingToDevicePixels() const final
     {
-        return rect;
+        auto scale = WebCore::GraphicsContextCG::scaleForRoundingToDevicePixels().value_or(std::make_pair<float>(1, 1));
```

Comment:
if someone is going to ask about the make_pair, the answer is that `{ 1, 1 }` results in a "could not deduce type" error.